src/pages/home/_platform_selector.tsx

```diff
@@ -0,0 +1,185 @@
+import React from 'react'
+import styled from 'styled-components'
+import { StyledLink, Text } from 'components/elements'
+import device from 'themes/device.js'
+
+export type Platform = {
+    title: string
+    icon: string
+    description: string
+    learn_more_link: string
+    download_links?: {
+        Browser?: string
+        AppStore?: string
+        GooglePlay?: string
+        Linux?: string
+        APK?: string
+    }
+}
+
+type PlatformSelectorProps = {
+    platforms: Array<Platform>
+    selected_index: number
+    selectIndex: React.Dispatch<React.SetStateAction<number>>
+}
+
+const ContentContainer = styled.div`
+    display: grid;
+    grid-template-rows: 1fr 1fr 1fr 3fr 1fr 1fr 1fr 1fr;
+    @media ${device.tabletL} {
+        transform: scale(0.8);
+    }
+`
+
+const UnselectedPlatform = styled.div`
+    align-self: start;
+    display: flex;
+    align-items: center;
+    width: 160px;
+    margin: 2rem;
+    margin-left: 0;
+    cursor: pointer;
+    img {
+        width: 32px;
+        height: 32px;
+        margin-right: 1rem;
+        opacity: 0.3;
+    }
+
+    @media ${device.laptopM} {
+        transform: scale(0.9);
+        margin-left: 0;
+    }
+`
+
+const SelectedPlatform = styled.div`
+    grid-row: 4;
+    display: flex;
+    align-items: flex-start;
+    width: 384px;
+    height: 200px;
+    padding-top: 1.6rem;
+    margin: 2rem 0;
+    background: #ffffff;
+    box-shadow: 0px 16px 20px rgba(131, 131, 131, 0.15), 0px 0px 20px rgba(131, 131, 131, 0.15);
+    border-radius: 6px;
+    @media ${device.laptopM} {
+        width: 345px;
+        height: 180px;
+        ${Text} {
+            font-size: 1.6rem;
+            line-height: 2.2rem;
+        }
```

Comment:
Try to avoid using selectors like this `${Text}` as it will override any responsive props passed to the component. These can be passed in as props instead.